doc/source/composites.rst

```diff
@@ -12,7 +12,7 @@ the key/values of the DataID used to identify it.
 Built-in Compositors
 ====================
 
-.. py:currentmodule:: satpy.composites
+.. py:currentmodule:: satpy.composites.core
```

Comment:
I *think* since you specify all API objects with a full path now then this line can be removed.

Reply:
I *thought* I removed it after changing to the full paths, but I guess I just removed the original one after I noticed how many times I'd need to use the `currentmodule` thing. (And there were also some places where there were two classes that were moved to different modules.)